bitnami/external-dns/values.yaml

```diff
@@ -41,7 +41,7 @@ sources:
   # - contour-httpproxy
 
 ## DNS provider where the DNS records will be created. Available providers are:
-## - alibabacloud, aws, azure, azure-private-dns, cloudflare, coredns, designate, digitalocean, google, infoblox, rfc2136, transip
+## - alibabacloud, aws, azure, azure-private-dns, cloudflare, coredns, designate, digitalocean, google, hetzner, infoblox, rfc2136, transip
```

Comment:
I think this change should be done in the values-production too.

Reply:
I agree, updated the PR with this change.